x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/overlay.tsx

```diff
@@ -47,7 +47,14 @@ export const NodeContextPopover = ({
   openAlertFlyout,
 }: Props) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  const tabConfigs = [MetricsTab, LogsTab, ProcessesTab, PropertiesTab, AnomaliesTab, OsqueryTab];
+  const tabConfigs = [
+    MetricsTab,
+    LogsTab,
+    ProcessesTab('infraNodeTabsProcesses'),
```

Comment:
Same as another comment. Or maybe use a spread operator and keep `ProcessesTab` with the bare minimum necessary to render a tab?

```
{ ...ProcessesTab, 'data-subj-test': infraNodeTabsProcesses }
```

Since the other tab representations are not functions but objects - I don't know why their names are capitalized either, as they are not components or classes - having this `ProcessTab` as a function seems a little bit confusing to me.